### PR TITLE
Minimum Version Changes

### PR DESCRIPTION
**Pull request by mitchelsellers**
Replaces #296

### CODE REVIEW

File: content/guides/tutorials/server/requirements/index.md [hunk 1]
```diff
@@ -18,10 +18,12 @@ The following are required for DNN Platform 9.0 and DNN Evoq 9.0.
 *   Windows Server 2008 R2
```
Comment (bdukes) [inline content/guides/tutorials/server/requirements/index.md]: We may want to note that SP1 is required for Windows Server 2008 R2 (per https://docs.microsoft.com/en-us/dotnet/framework/get-started/system-requirements#supported-server-operating-systems)

File: content/guides/tutorials/server/requirements/index.md [hunk 2]
```diff
@@ -53,23 +55,24 @@ The following are required for DNN Platform 9.0 and DNN Evoq 9.0.
...
+For contributing to DNN Platform:
 
-*   Visual Studio 2015
 *   Visual Studio 2017
 *   Visual Studio 2019
 
+
 ## Recommended Environment
 
 *   Windows Server 2012 R2
```
Comment (sleupold) [inline content/guides/tutorials/server/requirements/index.md]: Shouldn't we recommend using latest version (2019)?

File: content/guides/tutorials/server/requirements/index.md [hunk 3]
```diff
@@ -53,23 +55,24 @@ The following are required for DNN Platform 9.0 and DNN Evoq 9.0.
...
 *   IIS 8.5
-*   SQL Server 2014 Developer Edition
-*   **For developers:** Visual Studio 2015
+*   SQL Server 2014 or Later
+*   **For developers:** SQL Server 2014 Developer Edition or Express
```
Comment (sleupold) [inline content/guides/tutorials/server/requirements/index.md]: shouldn't it stage "SQL Server 2014/2016/2019 Developer Edition or Express?

File: content/guides/tutorials/server/requirements/index.md (hunk 1 above)
Comment (mitchelsellers) [reply content/guides/tutorials/server/requirements/index.md]: Updated

File: content/guides/tutorials/server/requirements/index.md (hunk 2 above)
Comment (mitchelsellers) [reply content/guides/tutorials/server/requirements/index.md]: I updated this, as actually really the latest version possible is the best recommendation

File: content/guides/tutorials/server/requirements/index.md (hunk 3 above)
Comment (mitchelsellers) [reply content/guides/tutorials/server/requirements/index.md]: Adjusted htis too

**Pull request merged**